commit 79370da601b9a51bbbb5001c246c6daad3bcf53d (refs/changes/16/3816/2)
Author: Jason King <jason.king@joyent.com>
Date:   2018-04-30T10:00:41-05:00 (1 year, 5 months ago)
    
    OS-6890 .WAIT doesn't work as an actual target in varpd
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/usr/src/lib/varpd/Makefile b/usr/src/lib/varpd/Makefile
index daa849572e..a0e4537ab9 100644
--- a/usr/src/lib/varpd/Makefile
+++ b/usr/src/lib/varpd/Makefile
@@ -10,10 +10,10 @@
 #
 
 #
-# Copyright 2015 Joyent, Inc.
+# Copyright 2018 Joyent, Inc.
 #
 
-SUBDIRS = libvarpd .WAIT direct files svp
+SUBDIRS = libvarpd direct files svp
 
 all :=		TARGET = all
 clean :=	TARGET = clean
@@ -26,6 +26,7 @@ lint :=		TARGET = lint
 .KEEP_STATE:
 
 all clean clobber install install_h check lint: $(SUBDIRS)
+direct files svp: libvarpd
 
 $(SUBDIRS):  FRC
 	@cd $@; pwd; $(MAKE) $(TARGET)
